### PR TITLE
Introduce better README

### PR DESCRIPTION
**Pull request by lucemans**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,148 @@
...
+
+```sh
+pnpm add permissio
+```
+
+## Usage
+
+Using Permissio simplifies permission manipulation and storage through intelligent use of bitfields.
+
+At its core you will be able to create an `enum` for your permissions, so long as the key of which is an integer.
```
Comment (svemat01) [inline README.md]:
```suggestion
At its core, you will be able to create an `enum` for your permissions, so long as the value of which is an integer.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,148 @@
...
+console.log(hasPermission(steve, Permissions.LIST)); // true
+console.log(hasPermission(steve, Permissions.DELETE)); // true
+console.log(hasPermission(steve, Permissions.USER_CREATE)); // false
+```
+
+## Documentation
+
+### hasPermission
+
+Gathering wether permissiondata contains a certain permission is as easy as follows:
```
Comment (svemat01) [inline README.md]:
```suggestion
Gathering whether permissiondata contains a certain permission is as easy as follows:
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,148 @@
...
+
+or if you prefer to use the `pnpm` package manager:
+
+```sh
+pnpm add permissio
+```
+
+## Usage
+
+Using Permissio simplifies permission manipulation and storage through intelligent use of bitfields.
```
Comment (svemat01) [inline README.md]:
```suggestion
Using Permissio simplifies permission manipulation and storage through the intelligent use of bitfields.
```

**Review by svemat01 — CHANGES_REQUESTED** [review]

**Review by svemat01 — CHANGES_REQUESTED** [review]
Maybe explain how enums only exists in typescript and the alternative solution on plain javascript

**Review by svemat01 — APPROVED** [review]
Javascript clarification was agreed between myself and @lucemans to be extended on / created in a new PR.

**Review by Antony1060 — APPROVED** [review]
LGTM

**Pull request merged**